instana/instrumentation/aiohttp/server.py

```diff
@@ -49,18 +73,23 @@ async def stan_middleware(request, handler):
                 response.headers['Server-Timing'] = "intid;desc=%s" % scope.span.context.trace_id
 
             return response
-        except Exception as e:
-            logger.debug("aiohttp stan_middleware", exc_info=True)
-            if scope is not None:
-                scope.span.set_tag("http.status_code", 500)
-                scope.span.log_exception(e)
+
+        except aiohttp.web_exceptions.HTTPError as exc:
+            handle_aiohttp_exception(scope, exc, exc.status_code)
+            raise
+
+        except asyncio.CancelledError as e:
```

Comment:
I think we no longer need any of our old change except for this `asyncio.CancelledError` handling. But we can no longer put it here, we need to put in the exception above in line 61. set a variable to mark that we want to skip the error like "skip_error = True" and add that checking in line 66. Other than that i think we can take all the change from their latest version.